onnx/test/shape_inference_test.py

```diff
@@ -189,48 +238,94 @@ def _identity_prop(self, op: str, **kwargs: Any) -> None:
             graph, [make_tensor_value_info("y", TensorProto.FLOAT, (30, 4, 5))]
         )
 
+    def get_caller_function_name(self):
+        """Return the function name that calls this get_caller_function_name."""
+        return inspect.stack()[1][3]
+
+    def test_no_op(self) -> None:
```

Comment:
Prefer to include the expected behavior when possible:

test_xxx_raises_key_error

or something more accurate. Here an everywhere else where appropriate. 

————-

I would include the expected behavior as part of the test method name. To make what a test is testing clear: `test_<method or function name>_<expected behavior>_[when_<condition>]`.

e.g. `test_method_x_raises_error_when_dims_is_not_a_sequence`

Reply:
Thanks for the detail explanation. I will try to following this format in the new test cases.
These two test cases are deleted.